### PR TITLE
add transfer credits page

### PR DESCRIPTION
**Pull request by moAlobaidi**
This PR is in tandem with this [PR](https://github.com/AleoHQ/aleo-docs/pull/214).

### CODE REVIEW

File: documentation/testnet/getting_started/04_transfer_credits.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+### `transfer_public_to_private`
+
+The `transfer_public_to_private` function is used to convert a public credit into a private credit.
+
+## Function Usage
+
+To use these functions, one needs to craft a `credits.aleo` program execution. Here is an example:
+
+```
+snarkos developer transfer <AMOUNT_TO_TRANSFER> --input-record <INPUT_RECORD> --recipient <RECIPIENT_ADDRESS> --private-key <PRIVATE_KEY> --query "<http://localhost:3030>" --broadcast "<http://localhost:3030/testnet3/transaction/broadcast>"
```
Comment (collinc97) [inline documentation/testnet/getting_started/04_transfer_credits.md:32]: Can you confirm wether or not this command takes `--input-record` or `--input_record`
same for the private key.

setting the endpoints to http://localhost:3030 assumes that the user has already started a snarkos node in addition to installing the CLI. Can you clarify that those steps are required or link to the documentation that you copied this line from?

**Review by collinc97 — CHANGES_REQUESTED** [review]

File: documentation/testnet/getting_started/04_transfer_credits.md (hunk 1 above)
Comment (moAlobaidi) [reply documentation/testnet/getting_started/04_transfer_credits.md:32]: @collinc97 Good catch! Will do.

Comment (moAlobaidi) [reply documentation/testnet/getting_started/04_transfer_credits.md:32]: Just confirmed the correct usage of the command and will update docs to match. 

Screen Shot 2023-06-26 at 12 23 14 PM

**Review by christianwooddell — APPROVED** [review]

File: documentation/testnet/getting_started/02_deploy_and_execute.md [hunk 2]
```diff
@@ -182,7 +182,7 @@ Transfer credits with a `credits.aleo` program execution.
 
 ##### Arguments:
 
-- `input_record` - The record used to craft the transfer
+- `input-record` - The record used to craft the transfer
```
Comment (collinc97) [inline documentation/testnet/getting_started/02_deploy_and_execute.md:185]:
```suggestion
- `input_record` - The record used to craft the transfer
```
I believe these are underscores https://github.com/AleoHQ/snarkOS/blob/a7cf1b4cb4dab5e66dee6f7b68ab2981af0c2690/cli/src/commands/developer/transfer_private.rs#L31

Can you the commands and double check?

**Review by collinc97 — CHANGES_REQUESTED** [review]

File: documentation/testnet/getting_started/02_deploy_and_execute.md (hunk 2 above)
Comment (moAlobaidi) [reply documentation/testnet/getting_started/02_deploy_and_execute.md:185]: Yes for sure, I tested the commands earlier with snarkOS on the testnet3 branch, see screenshot [here](https://github.com/AleoHQ/welcome/pull/221#discussion_r1242658658). I was only able to execute the commands with "-" not "_" on my end.

Comment (collinc97) [reply documentation/testnet/getting_started/02_deploy_and_execute.md:185]: great thanks for checking

File: documentation/testnet/getting_started/02_deploy_and_execute.md [hunk 3]
```diff
@@ -204,9 +204,10 @@ Transfer credits with a `credits.aleo` program execution.
 
 ```
 snarkos developer transfer <AMOUNT_TO_TRANSFER> --input-record <INPUT_RECORD> --recipient <RECIPIENT_ADDRESS>  --private-key <PRIVATE_KEY> --query "<http://localhost:3030>" --broadcast "<http://localhost:3030/testnet3/transaction/broadcast>"
-
 ```
 
+Note: Replace `transfer` with `transfer-private`, `transfer-public`, `-- transfer-private-to-public`,
+or `-- transfer_public_to_private` based on your use case.
```
Comment (collinc97) [inline documentation/testnet/getting_started/02_deploy_and_execute.md:210]:
```suggestion
Note: Replace `transfer` with `transfer-private`, `transfer-public`, `-- transfer-private-to-public`,
or `-- transfer_public_to_private` based on your use case.
```
Based on our previous discussion I believe `-- transfer-public-to-private` is correct?

**Review by collinc97 — CHANGES_REQUESTED** [review]

File: documentation/testnet/getting_started/02_deploy_and_execute.md (hunk 3 above)
Comment (moAlobaidi) [reply documentation/testnet/getting_started/02_deploy_and_execute.md:210]: Neither of those seem to work.. 
Screen Shot 2023-06-28 at 1 20 48 PM

I'm on the latest testnet3 branch and am getting these errors when trying to run any of the transfer commands other than `transfer-private`.

Just wanted to make a note of that before committing your suggestion, but assuming that is correct due to consistency.

Comment (collinc97) [reply documentation/testnet/getting_started/02_deploy_and_execute.md]: Thanks for checking, you are correct those flags do not exist.

**Comment by collinc97** [comment]
I am going to close this PR. Thank you for your work @moAlobaidi.
Merging this would partially document the snarkos developer command and leave our documentation incomplete. 
I think it also makes sense to wait until we have a more clear path for people to manage transfers.

**Pull request closed without merging**